tensorflow/core/kernels/data/map_dataset_op.cc

```diff
@@ -77,8 +80,8 @@ class MapDatasetOp : public UnaryDatasetOpKernel {
 
     std::unique_ptr<IteratorBase> MakeIteratorInternal(
         const string& prefix) const override {
-      return absl::make_unique<Iterator>(
-          Iterator::Params{this, strings::StrCat(prefix, "::Map")});
+      return absl::make_unique<Iterator>(Iterator::Params{
+          this, strings::StrCat(prefix, "::", kMapDatasetOpName)});
```

Comment:
FYI, this change will make the prefix different with before (e.g. `Iterator::Map` --> `Iterator::MapDataset`).

Reply:
This should be avoided because I am aware of tools that parse and analyze the labels (which show up in CPU profile traces).

Reply:
Alright, it is used in exactly one place (in dataset.cc) and I think we can replace it with a string based on `dataset_type_` once your PR is submitted.